proto/anpr_camera.proto

```diff
@@ -0,0 +1,89 @@
+syntax = "proto3";
+
+package smartcore.bos;
+
+option go_package = "github.com/vanti-dev/sc-bos/pkg/gen";
+
+import "google/protobuf/field_mask.proto";
+import "google/protobuf/timestamp.proto";
+
+
+// AnprCameraApi describes the capability to retrieve detection events from ANPR cameras.
+service AnprCameraApi {
+  rpc ListAnprEvents(ListAnprEventsRequest) returns (ListAnprEventsResponse) {}
```

Comment:
This is OK but have a look at e.g. the AirTemperatureHistory service which might be a better thing to replicate.
- It's a separate service so we don't require everything that implements AnprCameraApi to store historical records.
- It supports a specific time slice for history queries.